index.html

```diff
@@ -8,33 +8,179 @@
+                    <p>Canopy</p>
+                    <ul class="dev">
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>
+                        <button class="html">html</button>
+                    </li>
+                    <li>
+                        <button class="css">css</button>
+                    </li>
+                    <li >
+                        <button class="javascript">javascript</button>
+                    </li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div>
+            <div class="tonic-2">
+                <img src="images/tonic.png" alt="">
+                <p class="tonic">Tonic</p>
+                <div class="canopy">
+                    <p>Canopy</p>
+                    <ul class="dev">
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>
+                        <button class="html">html</button>
+                    </li>
+                    <li>
+                        <button class="css">css</button>
+                    </li>
+                    <li >
+                        <button class="javascript">javascript</button>
+                    </li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div>
+            <div class="multi-post-stories-2-container">
+                <img src="images/multi-post-stories-2.png" alt="">
+                <p class="tonic">Multi-post stories</p>
+                <div class="canopy">
+                    <p>Canopy</p>
+                    <ul class="dev">
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>
+                        <button class="html">html</button>
+                    </li>
+                    <li>
+                        <button class="css">css</button>
+                    </li>
+                    <li >
+                        <button class="javascript">javascript</button>
+                    </li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div> 
+        </section>
+        <section>
+            <div class="about-container">
+                <a class="about">About Myself</a>
```

Comment:
- [ ] Well, the `a` tag is used for s specific reason of linking, since you are using it to wrap the title and it does not have any link to redirect to, please change it and use an h2....5 tags instead.